### PR TITLE
Implement current API as per O-RAN Spec

### PR DESCRIPTION
**Pull request by aneeshkp**
calling currentstate via api with endpoint /resource/CurrentState
This PR only has changes to support CurrentState call from HTTP only.
AMQ implementation will be in separate PR 

Signed-off-by: Aneesh Puttur <aneeshputtur@gmail.com>

### CODE REVIEW

File: pkg/protocol/http/http.go [hunk 1]
```diff
@@ -203,13 +180,52 @@ func (h *Server) Start(wg *sync.WaitGroup) error {
...
+	}).Methods(http.MethodGet)
+
+	r.HandleFunc("/health", func(w http.ResponseWriter, req *http.Request) {
 		_ = json.NewEncoder(w).Encode(map[string]bool{"ok": true})
 	})
 
 	r.Handle("/event", eventHandler)
 	r.Handle("/subscription", subscriptionHandler)
-	r.Handle("/status", statusHandler)
+	//r.Handle("/status", statusHandler)
```
Comment (nishant-parekh) [inline pkg/protocol/http/http.go:228]: this can be removed correct?

Comment (aneeshkp) [reply pkg/protocol/http/http.go:228]: yes

**Review by nishant-parekh — APPROVED** [review]
/hold

**Comment by nishant-parekh** [comment]
holding for Jack to review before we merge

**Pull request merged**